cmd/spire-agent/cli/run/run.go

```diff
@@ -276,8 +281,13 @@ func validateConfig(c *config) error {
 		return errors.New("trust_domain must be configured")
 	}
 
-	if c.Agent.TrustBundlePath == "" {
-		return errors.New("trust_bundle_path must be configured")
+	switch {
+	case c.Agent.InsecureBootstrap:
+		if c.Agent.TrustBundlePath != "" {
+			return errors.New("trust_bundle_path must be not be configured when insecure_bootstrap is set")
```

Comment:
I could see a situation here in which someone is using a default/templated config that has some value set for trust bundle path (possibly even an old/expired bundle), but then passes insecure bootstrap cli flag to override it... considering that possibility, I think we'd be better off logging a warning when both are set, stating that we will ignore trust_bundle_path.

Perhaps that could be done in agent/attestor or something? Not sure what is most appropriate in that case

Reply:
Hmm, that's fair. I'll log in the attestor code.